recipe/meta.yaml

```diff
@@ -18,6 +18,7 @@ requirements:
     - __unix
     - python >=3.8,<3.10
     - pip
+    - setuptools
```

Comment:
Did the same thing here as for cylc-flow

https://github.com/conda-forge/cylc-flow-feedstock/pull/87/files#r1757174329

Reply:
I think it's missing from here too: https://github.com/conda-forge/cylc-uiserver-feedstock/blob/3f32ba188063b3143f5042f5df2ea4f71b801a66/recipe/meta.yaml#L106-L115

Reply:
Agreed.